src/utils/__tests__/paths.spec.ts

```diff
@@ -105,4 +109,14 @@ describe('paths', () => {
       console.log(`Skipping platform-specific test on unknown platform ${ os.platform() }`);
     }
   });
+
+  it('lima should be in one of the main subtrees', () => {
+    const pathsToDelete: Array<string> = [paths.cache, paths.appHome, paths.config, paths.logs];
```

Comment:
```suggestion
    const pathsToDelete = [paths.cache, paths.appHome, paths.config, paths.logs];
```
Since all elements are `string`, TypeScript can deduce that this is of type `string[]`.
(I also notice that you tend to use `Array<T>` over `T[]`; any particular reason for that?)

Reply:
I prefer `Array<T>` because it works with any type `T`.  `S[]` doesn't work with complex types